rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java

```diff
@@ -53,9 +53,6 @@
 
     private int beginIndex = 0;
 
-    public ListJobCommand() {
```

Comment:
- We provide whether the index and offset or the limit (max number of jobs), and we retrieve the jobs. Those arguments match the position of jobs in the list, and not their IDs.
- The default constructor ListJobCommand() is obsolete, that's why it was removed